packages/core/src/routes/sso-connector/index.ts

```diff
@@ -248,18 +251,20 @@ export default function singleSignOnRoutes<T extends AuthedRouter>(...args: Rout
       const {
         params: { id },
         body,
-        query: { partialValidateConfig },
+        query: { partialValidateConfig, shouldReplace },
       } = ctx.guard;
 
       const { providerName, config } = await getSsoConnectorById(id);
 
       // Merge with existing config and revalidate
       const parsedConfig = parseConnectorConfig(
         providerName,
-        {
-          ...config,
-          ...body,
-        },
+        yes(shouldReplace)
+          ? body
+          : {
+              ...config,
+              ...body,
+            },
```

Comment:
replace should call `PATCH ${pathname}/:id` endpoint with { config } body ?

Reply:
Good call, use `PATCH ${pathname}/:id` instead.